css/main.css

```diff
@@ -12,6 +12,7 @@
 
 body {
     display: block;
+    min-width: 360px;
```

Comment:
I think most phone are 320px or 360px so maybe target 320px here? Currently with this at 320px it introduces side scrolling which looks a bit weird.